x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.test.tsx

```diff
@@ -331,9 +331,11 @@ describe('Policy Details', () => {
         expect(tooltip).toHaveLength(0);
       });
 
-      it('ransomware card is hidden', () => {
+      it('ransomware card is hidden and locked card is shown instead', () => {
```

Comment:
Could be its own test. One makes sure ransomware is not visible, another to make sure lock is visible.

If it's possible to accidentally write a bug into the frontend such that _both_ are visible, then having one test for each, so only the relevant one fails is nice.

Reply:
As it stands, looks like the ternary makes it explicitly either-or.

But that may not always be the case

Reply:
kk